src/modules/project/pages1/projectFunding/views/fundingDetails/sections/FundingDetailsUserEmail.tsx

```diff
@@ -1,49 +1,195 @@
-import { Input } from '@chakra-ui/react'
+import { FormControl, FormErrorMessage, Input, InputGroup, InputRightElement, Switch } from '@chakra-ui/react'
+import { debounce } from 'lodash'
+import { useCallback, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
+import { BsFillCheckCircleFill, BsFillXCircleFill } from 'react-icons/bs'
```

Comment:
We've stopped using the icons from anything but react-icons/pi, which is the Phosphor Icons package. We're actively replacing other icons any touched portion of the code.